docs/guides/sqlite3_db_with_concord.md

```diff
@@ -0,0 +1,259 @@
+
+```c
+query = sqlite3_mprintf("INSERT INTO concord_guides(user_id, guild_id) values(%"PRIu64", %"PRIu64");", msg->author->id, msg->guild_id);
+rc = sqlite3_exec(db, query, NULL, NULL, &msgErr);
+
+sqlite3_free(query);
+
+if (rc != SQLITE_OK) {
+  log_fatal("[SQLITE] Something went wrong while inserting values into concord_guides table. [%s]", msgErr);
+  if (sqlite3_close(db) != SQLITE_OK) {
+        log_fatal("[SQLITE] Failed to close sqlite db. [%s]", sqlite3_errmsg(db));
+        abort();
+    }
+}
+
+sqlite3_close(db);
+```
+
+Pretty similar, isn't it? The difference is, the command now is "`INSERT INTO concord_guides(user_id, guild_id) values(%"PRIu64", %"PRIu64");`".
+
+The `user_id, guild_id` inside the () are the same that we put while creating the table, **it MUST match the one put when we created it**, after, in the () of the values we defined the value as the ID of the author of the command and the ID of the guild that the command has been executed.
+
+After it, we did the same thing we did when creating the table, freed the generated query, verified if the status code of the `sqlite3_exec` is not equal to `SQLITE_OK`, if it's not, log a fatal error and closes the DB, if it fails to close the DB, just abort. (NOTE: If you want a better explanation, see `Creating a sqlite3 table` above)
+
+### Reading the saved records
+
+And finally, after saving the records, we sometime will wish to read these records. This is a little more complicated than creating the table and saving records. See the following example:
+
+```c
+sqlite3_stmt *stmt = NULL;
+
+query = sqlite3_mprintf("SELECT user_id FROM concord_guides WHERE guild_id = %"PRIu64";", msg->guild_id);
+rc = sqlite3_prepare_v2(db, query, -1, &stmt, NULL);
+
+if ((rc = sqlite3_step(stmt)) != SQLITE_ERROR) {
+    printf("Sucessfully read the record: %lld.\n", sqlite3_column_int64(stmt, 0));
+}
+
+if (sqlite3_finalize(stmt) != SQLITE_OK) {
+    log_fatal("[SQLITE] Error while executing function sqlite3_finalize.");
+    abort();
+}
+
+sqlite3_close(db);
+```
+
+Here, instead of the `sqlite3_exec` (literally the `sqlite3_prepare_v2` wrapper), we are using the `sqlite3_prepare_v2` for better control of it.
+
+Now, the command is "`SELECT user_id FROM user_voice WHERE guild_id = %"PRIu64";`", where it wants to get the value of user_id, using as a search parameter the guild_id also saved before.
+
+After using the `sqlite3_mprintf` to generate the query, and preparing the statement with `sqlite3_prepare_v2`, we use the `sqlite3_step` that **evaluates the statement**, and if it returns SQLITE_ROW, it is going to successfully printf the records with `sqlite3_column_int64` (NOTE: **You may wish to use other `sqlite3_column_*` to get other types of records, like TEXT**).
+
+### Deleting records
+
+After creating a table, saving records, and reading the records, you may wish to delete those records, you are probably not wanting to stay with them forever.
```

Comment:
"After creating a table, saving records, and reading the records, you may wish to delete those records."

Reply:
Oh, thanks